### PR TITLE
"shared" template tier not worikng for sideboxes

### PR DESCRIPTION
**Pull request by Zen4All**
While testing the shared feature on side boxes I noticed that the files in include/modules/sideboxes/shared were ignored. I think this is because there is a condition missing in the column_left.php, column_right.php, and column_single.php.
After adding the conditions to the files the php files in the includes/modules/sideboxes/shared folder are used, when present.

[Review on Reviewable](https://reviewable.io/reviews/zencart/zc-v1-series/660)

### CODE REVIEW

**Comment by Zen4All** [comment]
missing files

File: admin/layout_controller.php [hunk 1]
```diff
@@ -29,7 +30,22 @@
...
+      if (!in_array($file, $dir_check, TRUE)) {
+        if (preg_match('~^[^\._].*\.php$~i', $file) > 0) {
+          $directory_array[] = $file;
+        }
+      }
+    }
+  }
+  $dir->close();
+}
+sort($directory_array);
```
Comment (drbyte) [inline admin/layout_controller.php]: I'd like to see these lines 33-48 indented 2 spaces to match the surrounding lines.

File: admin/layout_controller.php [hunk 2]
```diff
@@ -208,16 +224,34 @@
...
+
+                <td class="dataTableContent" align="right"><?php echo ( (file_exists($boxes_directory . $column_controller->fields['layout_box_name']) || file_exists($boxes_directory_shared . $column_controller->fields['layout_box_name']) || file_exists($boxes_directory_template . $column_controller->fields['layout_box_name'])) ? TEXT_GOOD_BOX : TEXT_BAD_BOX) ; ?>
+                  <?php
+                  if ((is_object($bInfo)) && ($column_controller->fields['layout_id'] == $bInfo->layout_id)) {
+                    echo zen_image(DIR_WS_IMAGES . 'icon_arrow_right.gif', '');
+                  } else {
+                    echo '<a href="' . zen_href_link(FILENAME_LAYOUT_CONTROLLER, 'page=' . $_GET['page'] . '&cID=' . $column_controller->fields['layout_id']) . '">' . zen_image(DIR_WS_IMAGES . 'icon_info.gif', IMAGE_ICON_INFO) . '</a>';
+                  }
+                  ?>
+                  &nbsp;</td>
```
Comment (drbyte) [inline admin/layout_controller.php]: I agree with the concept.  :+1: 

Haven't tested this page on a site yet...

**Comment by drbyte** [comment]
:+1:

**Comment by zcwilt** [comment]
:+1:

**Pull request merged**